x-pack/plugins/infra/public/components/waffle/lib/field_to_display_name.ts

```diff
@@ -24,19 +24,19 @@ export const fieldToName = (field: string, intl: InjectedIntl) => {
       id: 'xpack.infra.groupByDisplayNames.hostName',
       defaultMessage: 'Host',
     }),
-    'meta.cloud.availability_zone': intl.formatMessage({
+    'cloud.availability_zone': intl.formatMessage({
       id: 'xpack.infra.groupByDisplayNames.availabilityZone',
       defaultMessage: 'Availability Zone',
     }),
-    'meta.cloud.machine_type': intl.formatMessage({
+    'cloud.machine_type': intl.formatMessage({
```

Comment:
It's now `cloud.machine.type` and `cloud.project.id`. See https://github.com/elastic/beats/blob/master/dev-tools/ecs-migration.yml#L89

Reply:
Wow! That's a good catch. I'm glad we caught this now.